src/services/tryToLockCourtCase.ts

```diff
@@ -1,14 +1,16 @@
-import ResolutionStatus from "@moj-bichard7-developers/bichard7-next-core/build/src/types/ResolutionStatus"
-import { DataSource, UpdateResult } from "typeorm"
+import { DataSource, IsNull, MoreThan, Not, UpdateResult } from "typeorm"
 import CourtCase from "./entities/CourtCase"
+import User from "./entities/User"
+import { ResolutionStatus } from "types/ResolutionStatus"
 
 const tryToLockCourtCase = (
   dataSource: DataSource,
   courtCaseId: number,
-  userName: string,
-  lockException = true,
-  lockTrigger = true
+  user: User
 ): Promise<UpdateResult | Error> | Error => {
+  const lockException = user.canLockExceptions
```

Comment:
Can we just destructure this from user on line 9 so we don't need these variables?